infrastructure/php-container/src/.htaccess

```diff
@@ -97,16 +104,19 @@ DirectorySlash off
 
     # Direct static site requests to index files
     RewriteCond %{REQUEST_FILENAME} !-f
-    RewriteCond %{REQUEST_URI} /tc-report/_site/(.+)/?$
+    RewriteCond %{REQUEST_URI} ^/tc-report/_site/(.+)/?$
+    RewriteCond %{REQUEST_URI} !\.html$
     RewriteRule ^ tc-report/_site/%1/index.html [L]
 
     # Send tc-report/assets requests to correct folder
-    RewriteCond %{REQUEST_URI} /tc-report/assets/
+    RewriteCond %{REQUEST_URI} ^/tc-report/assets/
     RewriteRule ^tc-report/assets/(.*)$ tc-report/_site/assets/$1 [L]
 
     # Send all other requests to /tc-report/_site
-    RewriteCond %{REQUEST_URI} !/tc-report/_site
+    RewriteCond %{REQUEST_URI} !^/tc-report/_site
     RewriteRule ^(.*)/?$ tc-report/_site/$1 [L]
+
+    ErrorDocument 404 /404
```

Comment:
This really can't go live without translation, can it?

Reply:
Or a support page (which is the reason for https://github.com/GCTC-NTGC/gc-digital-talent/pull/4245#discussion_r994660338 😆 

I posed the question about what to do in slack and will try to remember to bring it up during stand up

Reply:
Translations completed in [add 404 French](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4245/commits/5fcde8caf3d76f9652c8b607f8c8a13a15b0c986)